src/config/constants.js

```diff
@@ -0,0 +1,227 @@
+        [TransactionType.VRF_KEY_LINK]: 'VRF Key Link',
+        [TransactionType.VOTING_KEY_LINK]: 'Voting Key Link',
+        [TransactionType.NODE_KEY_LINK]: 'Node Key Link',
+    };
+
+    static MosaicSupplyChangeAction = {
+        [MosaicSupplyChangeAction.Increase]: 'v_increase',
+        [MosaicSupplyChangeAction.Decrease]: 'v_decrease',
+    };
+
+    static NamespaceRegistrationType = {
+        [NamespaceRegistrationType.RootNamespace]: 'v_rootNamespace',
+        [NamespaceRegistrationType.SubNamespace]: 'v_subNamespace',
+    };
+
+    static AliasAction = {
+        [AliasAction.Link]: 'v_link',
+        [AliasAction.Unlink]: 'v_unlink',
+    };
+
+    static LinkAction = {
+        [LinkAction.Link]: 'v_link',
+        [LinkAction.Unlink]: 'v_unlink',
+    };
+
+    static AccountType = {
+        [AccountType.Unlinked]: 'Unlinked',
+        [AccountType.Main]: 'Main',
+        [AccountType.Remote]: 'Remote',
+        [AccountType.Remote_Unlinked]: 'Remote Unlinked',
+    };
+
+    static AccountKeyTypeFlags = {
+        [AccountKeyTypeFlags.Unset]: 'Unset',
+        [AccountKeyTypeFlags.Linked]: 'Linked',
+        [AccountKeyTypeFlags.VRF]: 'VRF',
+        [AccountKeyTypeFlags.Node]: 'Node',
+        [AccountKeyTypeFlags.All]: 'All',
+    };
+
+    static LockHashAlgorithm = {
+        [LockHashAlgorithm.Op_Sha3_256]: 'Sha3 256',
+        [LockHashAlgorithm.Op_Hash_160]: 'Hash 160',
+        [LockHashAlgorithm.Op_Hash_256]: 'Hash 256',
+    };
+
+    static MetadataType = {
+        [MetadataType.Account]: 'Account',
+        [MetadataType.Mosaic]: 'Mosaic',
+        [MetadataType.Namespace]: 'Namespace',
+    };
+
+    static ReceiptType = {
+        [ReceiptType.Harvest_Fee]: 'Harvest Fee',
+        [ReceiptType.LockHash_Created]: 'LockHash Created',
+        [ReceiptType.LockHash_Completed]: 'LockHash Completed',
+        [ReceiptType.LockHash_Expired]: 'LockHash Expired',
+        [ReceiptType.LockSecret_Created]: 'LockSecret Created',
+        [ReceiptType.LockSecret_Completed]: 'LockSecret Completed',
+        [ReceiptType.LockSecret_Expired]: 'LockSecret Expired',
+        [ReceiptType.Mosaic_Levy]: 'Mosaic Levy',
+        [ReceiptType.Mosaic_Rental_Fee]: 'Mosaic Rental Fee',
+        [ReceiptType.Namespace_Rental_Fee]: 'Namespace Rental Fee',
+        [ReceiptType.Mosaic_Expired]: 'Mosaic Expired',
+        [ReceiptType.Namespace_Expired]: 'Namespace Expired',
+        [ReceiptType.Namespace_Deleted]: 'Namespace Deleted',
+        [ReceiptType.Inflation]: 'Inflation',
+    };
+
+    static ResolutionType = {
+        [ResolutionType.Address]: 'Address',
+        [ResolutionType.Mosaic]: 'Mosaic',
+    };
+
+    static NetworkType = {
+        [NetworkType.MAIN_NET]: 'MAINNET',
+        [NetworkType.MIJIN]: 'MIJIN',
+        [NetworkType.MIJIN_TEST]: 'MIJIN TESTNET',
+        [NetworkType.TEST_NET]: 'TESTNET',
+        [NetworkType.PRIVATE_TEST]: 'PRIVATE TESTNET',
+        [NetworkType.PRIVATE]: 'PRIVATE',
+    };
```

Comment:
I think we should only keep `[NetworkType.MAIN_NET]` and `[NetworkType.TEST_NET]` the rest can be remove